app/controllers/mailgun_webhooks_controller.rb

```diff
@@ -10,7 +10,10 @@ def create_incoming_email
     clients = Client.joins(:intake).where(intakes: { email_address: sender_email })
     client_count = clients.count
     if client_count.zero?
-      archived_intake = Archived::Intake2021.where(email_address: sender_email).first
+      archived_intake = Client.archived_intake_models.find do |klass|
+        intake = where(email_address: sender_email).first
+        return intake if intake.present?
```

Comment:
looks the same as the one in hub/clients_controller before it got changed to a `break`. maybe should be extracted to a model on client (`find_archived_intake`? or something like that)